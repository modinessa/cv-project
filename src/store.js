import { configureStore } from "@reduxjs/toolkit";
import { navigationPanelReducer } from "./components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationPanelReducer,
	}
});
