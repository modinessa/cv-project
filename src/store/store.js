import { configureStore } from "@reduxjs/toolkit";
import navigationReducer from "../components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationReducer,
	}
});