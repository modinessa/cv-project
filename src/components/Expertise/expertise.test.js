import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';

import { store } from '../../store/store';
import { Expertise } from './index';

const TEST_DATA = [
	{
    date: '1900-2000', 
    info: {
      company: 'Test',
      job: 'Some Job',
      description: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Labore esse tempor nisi non mollit enim elit ullamco veniam elit duis nostrud. Enim pariatur ullamco dolor eu sunt ad velit aute.'
		}
  }
]

describe('Expertise', () => {
	it('renders component', () => {
		expect(renderer.create(
			<Provider store={store}>
				<Expertise data={TEST_DATA} />
			</Provider>
		)).toMatchSnapshot();
	})
})