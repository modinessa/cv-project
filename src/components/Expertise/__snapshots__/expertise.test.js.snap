// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expertise renders component 1`] = `
<section
  className="app-section"
>
  <h2>
    Experience
  </h2>
  <div
    className="app-experience-item"
  >
    <div
      className="app-experience-title"
    >
      <h3>
        Test
      </h3>
      <h4>
        1900-2000
      </h4>
    </div>
    <div
      className="app-experience-info"
    >
      <h3>
        Some Job
      </h3>
      <p>
        Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Labore esse tempor nisi non mollit enim elit ullamco veniam elit duis nostrud. Enim pariatur ullamco dolor eu sunt ad velit aute.
      </p>
    </div>
  </div>
</section>
`;
